packages/web/src/fonts/Fonts.tsx

```diff
@@ -11,6 +13,27 @@ interface Props {
   accessibilityRole?: 'button' | 'label' | 'link' | 'heading' | 'listitem'
 }
 
+interface TableProps {
+  style?: any
+  children?: any
+}
+
+export const TABLE = ({ style, children }: TableProps) => {
```

Comment:
moved these from the `Text` file in shared.  while also replace the one off style with our equivalent `fonts.legal` style